src/plugins/discord/models.js

```diff
@@ -64,7 +64,7 @@ export type User = {|
 
 export type GuildMember = {|
   +user: User,
-  +nick: string | null,
+  +nick: string | ?null,
```

Comment:
`string | ?null` is just a more confusing way of saying `?string`

Reply:
Ah yea, I forget in Flow that ? includes null and undefined. In TypeScript null and undefined have to be specified separately. Will fix